devtron-labs/dashboard
# feat: In CD add repo and branch for the corresponding commit in the title

# Description

CD Select history > Add repo and branch for the corresponding commit

<img width="916" alt="Screenshot 2023-03-02 at 12 14 48 AM" src="https://user-images.githubusercontent.com/61836271/222235316-caea22dd-0bb7-4107-a5cc-ee3daa7f483b.png">

Fixes [#AB2182](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/OSS%20adoption/Stories/?workitem=2182)

## Type of change


# How Has This Been Tested?
1. Go to Build & Deploy
2. Click on `Select` in deploy node
3. Go to the source info
4. check if the repo and the branch added



# Checklist:
